app/assets/javascripts/app/review_form.js

```diff
@@ -177,7 +182,7 @@ App.addChild('ReviewForm', _.extend({
     this.$state = this.$('#contribution_address_state');
     this.$phone = this.$('#contribution_address_phone_number');
     this.$zip = this.$('#contribution_address_zip_code');
-
+    this.$cpf = this.$('#payment_card_cpf');
```

Comment:
tu seta a variável pra esse mas n seta  pros outros campos pq?
`this.$('#payment_card_phone').mask('(99) 9999-99999');`
`this.$('#payment_card_birth').mask('99/99/9999');`
`this.$('#payment_card_date').mask('99/99');`